release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -87,6 +87,42 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-networking"]
 === Networking
 
+[id="ocp-4-13-networking-metrics"]
+==== Enhancements to networking metrics
+[cols="2,2",options="header"]
+.Enhancements to network metrics
+|====
+|Name | Help message
+|`egress_ips_rebalance_total`
```

Comment:
we should also include the namespace + subsystem for all these metrics so users see  the full metric name.